src/looprestoration_tmpl_16.rs

```diff
@@ -1109,6 +938,9 @@ unsafe extern "C" fn wiener_filter_neon(
     edges: LrEdgeFlags,
     bitdepth_max: libc::c_int,
 ) {
+    use crate::src::looprestoration::LR_HAVE_BOTTOM;
```

Comment:
Can't these be top-level imports?

Reply:
They can be, but they'd have to be conditionally imported to avoid warnings since they're only used within `#[cfg(arm)]` code. I figured importing them within the functions they're used was the cleaner approach, but I can make them top-level instead if you prefer.

Reply:
Ah, it's fine as is then.